src/test/java/com/deviceinsight/kafka/health/KafkaConsumingHealthIndicatorTest.java

```diff
@@ -64,15 +65,14 @@ public void kafkaIsDown() throws Exception {
 		final KafkaConsumingHealthIndicator healthIndicator =
 				new KafkaConsumingHealthIndicator(kafkaHealthProperties, kafkaProperties.buildConsumerProperties(),
 						kafkaProperties.buildProducerProperties());
-		healthIndicator.subscribeToTopic();
+		healthIndicator.subscribeAndSendMessage();
 
 		Health health = healthIndicator.health();
 		assertThat(health.getStatus()).isEqualTo(Status.UP);
 
 		shutdownKafka();
 
-		health = healthIndicator.health();
-		assertThat(health.getStatus()).isEqualTo(Status.DOWN);
+		Awaitility.await().untilAsserted(() -> assertThat(healthIndicator.health().getStatus()).isEqualTo(Status.DOWN));
```

Comment:
Does this have a default expiration or will it keep waiting forever? If so, I would like it to have a timeout so that our builds stop if something goes awry.

Reply:
By default Awaitility will wait for 10 seconds.